k4FWCore/scripts/k4run

```diff
@@ -111,27 +111,33 @@ def add_arguments(parser, app_mgr):
     return option_db
 
 
-class LoggingHandler(logging.StreamHandler):
-    def handleError(self, record):
-        # Re-raise SIGPIPE generated exception
-        excp = sys.exc_info()[1]  # from Python 3.11 can be replaced with sys.exception()
-        if isinstance(excp, BrokenPipeError):
-            raise excp
+def parse_log_level(level_str):
```

Comment:
Maybe this function is not needed since `set_log_level` checks and is called early on so if the level is wrong then it's going to fail quickly anyway.

Reply:
No clear preference from my side, I think this was there before I realized I had to handle `VERBOSE` as well. The advantage with the current version is that it is very clear for users where the error comes from. But maybe moving the second part of the error message to the `set_log_level` gives them the same information.

Reply:
I think I would remove, one less function to maintain.

Reply:
I always forget that `add_argument` takes a `choices` kwarg, so there is no need for custom handling. I have switched to that and added more information to the error message in `set_log_level`.

Reply:
This seems to have been lost in the rebasing / merge conflict resolution (i.e. the removal of the `parse_log_level`)

Reply:
Argh that was probably that the branch I had wasn't up to date with the remote, and since I saw most of the stuff I assumed it was. Do you still have the commit if you didn't pull?

Reply:
I'll just try it in a commit since it should be an easy change

Reply:
I can also try to do the rebase push back that commit. I still have it locally

Reply:
Ok, then I don't do anything :smile:

Reply:
:+1: Done.